### PR TITLE
twm: add page

### PR DESCRIPTION
**Pull request by owenvoke**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/twm.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`twm`
+
+- Connect to a specific X server:
+
+`twm -display {{display}}`
+
+- Only manage the default screen:
+
+`twm -s`
```
Comment (navarroaxel) [inline pages/common/twm.md:16]: Is there a long argument format?

File: pages/common/twm.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# twm
```
Comment (navarroaxel) [inline pages/common/twm.md:1]: Should this be moved to the `linux` platform directory?

File: pages/common/twm.md (hunk 1 above)
Comment (owenvoke) [reply pages/common/twm.md:16]: I wasn't able to find one. 😬

File: pages/common/twm.md (hunk 2 above)
Comment (owenvoke) [reply pages/common/twm.md:1]: It works in Mac and Linux, so I wasn't sure. But I guess it could be in the Mac and Linux directories instead. 🤷🏻

Comment (navarroaxel) [reply pages/common/twm.md:1]: I invoke to @sbrl to know what to do. lol

Comment (sbrl) [reply pages/common/twm.md:1]: If it works on more than 1 platform, then it goes in `common` @navarroaxel :slightly_smiling_face:

**Review by sbrl — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**